code/modules/surgery/organs/internal/lungs/_lungs.dm

```diff
@@ -794,6 +793,9 @@
 /obj/item/organ/internal/lungs/get_availability(datum/species/owner_species, mob/living/owner_mob)
 	return owner_species.mutantlungs
 
+#define SMOKER_ORGAN_HEALTH (STANDARD_ORGAN_THRESHOLD * 0.75)
+#define SMOKER_LUNG_HEALING (STANDARD_ORGAN_HEALING * 0.75)
```

Comment:
can you `#undef` these at the end of your file? the thing is that this file has a `_` prefix, so that means that the linters won't run on it because of mothblocks. this doesn't need to be accessed outside of this file though